app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -151,6 +151,17 @@ const processAuthorizationOptions = (item: PostmanAuth.Item | undefined, parentC
   return auth;
 };
 
+const getContentTypeForRawBody = (bodyType: string) => {
+  switch (bodyType) {
+    case "json":
+      return RequestContentType.JSON;
+    case "text":
+      return RequestContentType.RAW;
+    case "html":
+      return RequestContentType.HTML;
+  }
+};
+
```

Comment:
We should cover all possible content types here. 
this seems to be missing cases for `xml` and `javascript`. 

Will also need to consider how body from such imports will be visible in the code editor

Reply:
Request content-type can be anything ([refer](https://www.iana.org/assignments/media-types/media-types.xhtml)), we should cover the most used ones, WDYT @nsrCodes?

Also, since we support `Raw` format for request body, then sever should be able to handle by itself (confirm this), in this case we dont need to take any action on how it will be visible in code editor

Reply:
Request content-type can be anything ([refer](https://www.iana.org/assignments/media-types/media-types.xhtml)), we should cover the most used ones, WDYT @nsrCodes?

Also, since we support `Raw` format for request body, then sever should be able to handle by itself (confirm this), in this case we dont need to take any action on how it will be visible in code editor

Reply:
@rohanmathur91  I mentioned these specifically from the list of options provided by postman.
![CleanShot 2025-05-20 at 17 18 02@2x](https://github.com/user-attachments/assets/886882df-3a4e-4a74-aeb2-200fdf2b60b5)

Reply:
Got it.